README.md

```diff
@@ -1,126 +1,131 @@
-
-As mentioned previously, the KoP protocol handler is a plugin that can be installed to the Pulsar brokers.
-
-All what you need to do is to configure the Pulsar broker to run the KoP protocol handler as a plugin, that is,
-add configurations in Pulsar's configuration file, such as `broker.conf` or `standalone.conf`.
+## Set configuration for KoP
 
-1. Set the configuration of the KoP protocol handler.
+After you copy the .nar file to your Pulsar `/protocols` directory, you need to configure the Pulsar broker to run the KoP protocol handler as a plugin by adding configurations in Pulsar configuration file `broker.conf` or `standalone.conf`.
 
-    Add the following properties and set their values in Pulsar configuration file, such as `conf/broker.conf` or `conf/standalone.conf`.
-
-    KoP supports partitioned topics only. Therefore, you had better to set `allowAutoTopicCreationType` to `partitioned`. If it is set to `non-partitioned` by default, the topics that are automatically created by KoP are still partitioned topics. However, topics that are created automatically by Pulsar broker are non-partitioned topics.
-
-    | Property | Set it to the following value | Default value |
-    | :------- | :---------------------------- | :------------ |
-    | `messagingProtocols` | kafka | null |
-    | `protocolHandlerDirectory`| Location of KoP NAR file | ./protocols |
-    | `allowAutoTopicCreationType`| partitioned | non-partitioned |
-
-    **Example**
+1. Set the configuration of the KoP protocol handler in `broker.conf` or `standalone.conf` file.
 
     ```properties
     messagingProtocols=kafka
     protocolHandlerDirectory=./protocols
     allowAutoTopicCreationType=partitioned
     ```
 
-2. Set Kafka service listeners.
+    | Property | The value you need to set | Default value |
+    | :------- | :---------------------------- | :------------ |
+    | `messagingProtocols` | kafka | null |
+    | `protocolHandlerDirectory`| Location of KoP NAR file | ./protocols |
+    | `allowAutoTopicCreationType`| partitioned | non-partitioned |
 
-    > #### Note
-    > The hostname in listeners should be the same as Pulsar broker's `advertisedAddress`.
+    You need to set `allowAutoTopicCreationType` to `partitioned` since KoP only supports partitioned topics. If it is set to `non-partitioned` by default, the topics created automatically by KoP are still partitioned topics, yet topics created automatically by Pulsar broker are non-partitioned topics.
 
-    **Example**
+2. Set Kafka service listeners.
 
     ```properties
     listeners=PLAINTEXT://127.0.0.1:9092
     advertisedAddress=127.0.0.1
     ```
-3. Offset Management
 
-    Offset management for KoP is dependent on "Broker Entry Metadata" feature of Pulsar. So, you should set `brokerEntryMetadataInterceptors` to `org.apache.pulsar.common.intercept.AppendIndexMetadataInterceptor`.
+    > The hostname in listeners should be the same as Pulsar broker's `advertisedAddress`.
```

Comment:
```suggestion
```

From 2.8.0, this limit doesn't exist anymore.

Reply:
Got it, thank you, I'll remove it.